swap/src/asb/command.rs

```diff
@@ -210,6 +221,13 @@ pub struct RawArguments {
     )]
     pub json: bool,
 
+    #[structopt(
+        short,
+        long = "timestamp",
+        help = "Adds a timestamp to the log messages"
+    )]
+    pub timestamp: bool,
```

Comment:
I think the opposite default (timestamps enabled by default) would be better.

AFAIK, there is a structopt thing for doing negative flags!